lib/setup.js

```diff
@@ -633,6 +633,26 @@ function processCommand(command, args, params, callback) {
 
                     setup.setup((isFirst, isRedis) => {
                         if (isFirst) {
+                            const initialInstances = ['admin', 'discovery', 'info'];
```

Comment:
Didn't we talk about no longer installing discovery? I expect this change to make adapter tests take even longer. We probably need to address this, so integration tests no longer install any instances.

Reply:
discovery needed because install wizard needs it and it is part of the current "onbarding" ... if we need to enhance discovery to find mire stuff :-)

Fort tests you are right ... but not relevant because also currently no discovery instance is installed because dscovery is not existing (and so not resolving). So same for info. unless we test core package where we install both and here it is wanted